binarytree/__init__.py

```diff
@@ -16,6 +16,8 @@
     NodeReferenceError,
 )
 
+from binarytree.layout import _generate_svg
```

Comment:
Could you please rename module `layout` to `svg` and rename `_generate_svg` to `generate_svg`?